packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/AudienceView.tsx

```diff
@@ -25,12 +43,48 @@ export interface AudienceViewProps extends HasContainerId {
 export function AudienceView(props: AudienceViewProps): React.ReactElement {
 	const { containerId } = props;
 
-	// TODO: Post message requesting Audience summary
-	// TODO: Listen for Audience data updates
+	const messageRelay = React.useContext(MessageRelayContext);
```

Comment:
If you pull main, you can replace this call and subsequent `undefined` check with
```typescript
const messageRelay = useMessageRelay();
```

See #14564 for reference.

Reply:
With the import of `import { useMessageRelay } from "./MessageRelayContext";`, changed to `const messageRelay = useMessageRelay();`